src/container/TasksItem.js

```diff
@@ -1,15 +1,23 @@
 /* Created by IATS on 10.02.2018. */
 import React from 'react';
+import PropTypes from 'prop-types';
+
 
 export default class TasksItem extends React.Component {
+    static propTypes = {
+        TasksListThis: PropTypes.array.isRequired,
```

Comment:
Do not send TasksListThis in this component. You need only listValue and onItemRemove props.